### ISSUE TITLE
Why the loss function does not converge?

### ISSUE THREAD

**Issue by TSNeng** [megvii-research/CVPR2023-DMVFN#11]
I did just as readme.md wrote, only changed 'num_gpu' and 'nproc_per_node' to 4 cause i only have 4 gpus.
btw, shouldn't the argument ‘--local_rank' be '--local-rank'?
I would appreciate it if you could answer！

**Comment by TSNeng** [comment]
I think it maybe difference of the version of torch or other packages. Could you provide the specific version of packages. Thank you!

**Comment by huxiaotaostasy** [comment]
packages
---------------------
tensorboard             2.14.0
tensorboard-data-server 0.7.1
termcolor               2.3.0
thop                    0.1.1.post2209072238
tifffile                2023.8.25
timm                    0.9.6
torch                   1.11.0
torchaudio              0.11.0
torchstat               0.0.7
torchvision             0.12.0
tqdm                    4.66.1
typeguard               4.1.3
typing_extensions       4.7.1
tzdata                  2023.3
ujson                   5.8.0
urllib3                 1.26.16
websocket-client        1.6.2
Werkzeug                2.3.7
wheel                   0.38.4
wrapt                   1.15.0
yacs                    0.1.8
---------------------
I noticed that different numbers of gpu corresponded to different learning rates. I suggest you retrain the network and try to change the learning rate and weight decay.
